src/details/BookDetails.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import MarketplaceLink from './MarketplaceLink'
+import StarRating from './StarRating'
+import BookYearsReferenced from './BookYearsReferenced'
+
+class BookDetails extends Component {
+
+    createMarkup(book) {
+        return {__html: book.summary};
+    }
+
+    render() {
+        let book = this.props.book;
+        return (
+            <div className="row">
+                <div className="col-md-3 col-sm-12">
+                    <MarketplaceLink link={book.marketplaceLink}/>
+                </div>
+                <div className="col-md-9 col-sm-12">
+                    <h2 className="left-align">{book.title}</h2>
+                    <h3 className="left-align">{book.author}</h3>
+                    <StarRating rating={book.rating}/>
+                    <BookYearsReferenced start={book.start} end={book.end} />
+                    <div className="summary">
+                        <div dangerouslySetInnerHTML={this.createMarkup(book)} />
+                    </div>
+                </div>
+            </div>
+        )
+    }
+}
+
+BookDetails.defaultProps = {
```

Comment:
You should probably separate this out into its own file. keeping it in here is confusing